Networking/WebSockets/WebSocketProtocol.hh

```diff
@@ -337,7 +337,13 @@ namespace uWS {
                 auto*     start = reinterpret_cast<uint32_t*>(dst + headerLength);
                 uint32_t* end   = start + length / 4;
 
-                while ( start != end ) { *start++ ^= mask_i32; }
+                while ( start != end ) {
+                    uint32_t chunk;
+                    std::memcpy(&chunk, start, sizeof(chunk));
+                    chunk ^= mask_i32;
+                    std::memcpy(start, &chunk, sizeof(chunk));
+                    start++;
+                }
 
```

Comment:
Didn't it cause Xcode warning?

Reply:
If you mean the run-time memory warning which we discussed - this fixes that. Per Apple's documentation, `std::memcpy` automatically resolves the memory alignment issues which can be hit by plain assignment, so this is their recommended solution.

Reply:
I see.

Reply:
I cannot believe someone would unironically suggest this as a solution....adding two more memory copies?!